main/src/ca/uwaterloo/flix/language/ast/FinalAst.scala

```diff
@@ -233,6 +245,10 @@ object FinalAst {
 
   }
 
+  case class SelectChannelRule(sym: Symbol.VarSym, chan: FinalAst.Expression, exp: FinalAst.Expression)
+
+  case class SelectChannelDefault(exp: FinalAst.Expression)
```

Comment:
I did not realize that the default carries no other information. In that case, I would prefer just to inline it directly in SelectChannel, e.g. default: Option[FinalAst.Expression].

This applies to other ASTs aswell.